### PR TITLE
Commerce 6609 required changes

### PR DESCRIPTION
**Pull request by MahmoudAzzam**

### CODE REVIEW

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

File: modules/apps/dispatch/dispatch-talend-web/src/main/java/com/liferay/dispatch/talend/web/internal/display/context/DispatchTalendDisplayContext.java [hunk 1]
```diff
@@ -1,4 +1,41 @@
...
+	public DispatchTriggerMetadata getDispatchTriggerMetadata(
+		long dispatchTriggerId) {
+
+		return _dispatchTriggerMetadataProvider.getDispatchTriggerMetadata(
+			dispatchTriggerId);
+	}
+
+	private final DispatchTriggerMetadataProvider
+		_dispatchTriggerMetadataProvider;
+
```
Comment (igorbeslic) [inline modules/apps/dispatch/dispatch-talend-web/src/main/java/com/liferay/dispatch/talend/web/internal/display/context/DispatchTalendDisplayContext.java:41]: @MahmoudAzzam if you do not close Java source file with closing curly bracket compiler generates compile time error

Comment (igorbeslic) [reply modules/apps/dispatch/dispatch-talend-web/src/main/java/com/liferay/dispatch/talend/web/internal/display/context/DispatchTalendDisplayContext.java:41]: pay attention to it

**Comment by igorbeslic** [comment]
@MahmoudAzzam see https://github.com/liferay-commerce/liferay-portal/pull/2254

**Pull request closed without merging**